lib/miq_environment.rb

```diff
@@ -37,7 +37,7 @@ def self.supports_apache?
 
     def self.supports_systemd?
       return @supports_systemd unless @supports_systemd.nil?
-      @supports_systemd = is_appliance? && !is_container? && supports_command?('systemctl')
+      @supports_systemd = supports_command?('systemctl')
```

Comment:
don't we need the not-a-container check? I thought that was important.

Reply:
yeah see https://github.com/ManageIQ/manageiq/pull/21031/files#r573304624